app/views/foods/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>List of Food... yum!</h1>
+
+<p>Total hits for "<%= params[:q] %>": <%= @facade.total_hits %></p>
```

Comment:
If possible I'd limit your use of `params` in the view and rely on what is sent down from the controller overall, since the `params` object can change regardless of what the controller sends.